### ISSUE TITLE
`max-empty-lines` reports no run of empty lines whose breaks are spelled both ways

### ISSUE THREAD

**Issue by firefoxic** [stylelint-stylistic/stylelint-stylistic#586]
### What minimal example or steps are needed to reproduce the bug?

`max-empty-lines` looks for the Windows pair wherever the file holds one and for the line feed otherwise, and counts a run of empty lines as consecutive matches. A run spelling its breaks both ways is not consecutive to it, so it is neither reported nor written.

Measured with `@stylistic/max-empty-lines: 1` on `177f15b` and on the branch of #481, which changes none of these rows:

| input | without `--fix` | with `--fix` |
| --- | --- | --- |
| `a {\r\n\tb: c;\r\n\n\r\n\n}\r\n` — the pair and the feed taking turns | none | unchanged |
| `a {\r\n\tb: c;\n\n\n\n}\r\n` — a run of feeds in a file of pairs | none | unchanged |

Both files show three empty lines in front of the closing brace to an editor and to PostCSS's line counter, which reads a bare feed and a pair alike.

### What minimal configuration is needed to reproduce the bug?

```json
{
	"plugins": ["@stylistic/stylelint-plugin"],
	"rules": {
		"@stylistic/max-empty-lines": 1
	}
}
```

### What did you expect to happen?

The run is reported and collapsed like one spelled with a single kind of break.

### What actually happened?

Nothing is reported.

### Notes

Found by the review of #481, whose sweep carries such runs as an axis and found them blind on both sides — 2 835 rows, no warning. Whether a mixed run should be reported at all is a question for `linebreaks`, which is the rule about the spelling; but a rule that counts lines the way PostCSS counts them would report the count and leave the spelling alone.
